### ISSUE TITLE
Missing Aria Label Types for Pagination

### ISSUE THREAD

**Issue by dominikkaegi** [mbrn/material-table#1002]
**Describe the bug**
TS Type definition of Localization Pagination is missing types for the aria-labels e.g. `firstAriaLabel`

**To Reproduce**
1. Import Localization Type
2. Use it on an object and try to use firstAriaLabel

```js
import { Localization} from 'material-table';

const localization: Localization = {
  pagination: {
    firstAriaLabel: 'Erste Seite',
  },
};
```
```
Error
Object literal may only specify known properties, and 'firstAriaLabel' does not exist in type
```

**Expected behavior**
The aria labels should be defined as types and throw no tsc errors.

**Additional context**
The existing aria-label types can be found in the [docs](https://material-table.com/#/docs/features/localization).

**Comment by jayarjo** [comment]
PRs are welcome.

**Comment by dominikkaegi** [comment]
We can close it. I already created a PR and it is already merged 👍
